xbmc/video/tags/VideoTagLoaderNFO.cpp

```diff
@@ -0,0 +1,210 @@
+/*
+ *      Copyright (C) 2005-2015 Team Kodi
+ *      http://kodi.tv
+ *
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ *
+ *  This Program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Kodi; see the file COPYING.  If not, see
+ *  <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "VideoTagLoaderNFO.h"
+#include "FileItem.h"
+#include "filesystem/Directory.h"
+#include "filesystem/File.h"
+#include "filesystem/StackDirectory.h"
+#include "NfoFile.h"
+#include "video/VideoInfoTag.h"
+#include "utils/log.h"
+#include "utils/StringUtils.h"
+#include "utils/URIUtils.h"
+
+using namespace XFILE;
+
+CVideoTagLoaderNFO::CVideoTagLoaderNFO(const CFileItem& item,
+                                       ADDON::ScraperPtr info,
+                                       bool lookInFolder)
+  : IVideoInfoTagLoader(item, info, lookInFolder)
+{
+  if (m_info->Content() == CONTENT_TVSHOWS && m_item.m_bIsFolder)
+    m_path = URIUtils::AddFileToFolder(m_item.GetPath(), "tvshow.nfo");
+  else
+    m_path = FindNFO(m_item, lookInFolder);
+}
+
+bool CVideoTagLoaderNFO::HasInfo() const
+{
+  return !m_path.empty() && CFile::Exists(m_path);
+}
+
+CInfoScanner::INFO_TYPE CVideoTagLoaderNFO::Load(CVideoInfoTag& tag,
+                                                 bool prioritise,
+                                                 std::vector<EmbeddedArt>*)
```

Comment:
Missing `art` parameter name (and below too)

Reply:
It is called parameter anonymization and is done on purpose.

Reply:
cool